src/Playwright.Tests/PageRequestInterceptTests.cs

```diff
@@ -70,6 +70,9 @@ await route.FulfillAsync(new()
                     Response = response,
                     Status = 201,
                     Body = "",
+                    Headers = new Dictionary<string, string> {
```

Comment:
this fixes tests, was also made upstream the same way. 

Class got renamed to align it with file name which is common in c# and we do it everywhere else.